data/json/items/migration.json

```diff
@@ -20,6 +20,12 @@
     "replace": "battery_car"
   },
   {
+    "id": "gold_small",
+    "type": "MIGRATION",
+    "replace": "gold",
+    "charges": 1
```

Comment:
Does that mean one old gold bar (old save) is converted to one gold charges (new save)?

Gold and silver bars can currently be disassembled to 800 small units. This ratio should be preserved, if possible.